src/main/java/io/supertokens/config/CoreConfig.java

```diff
@@ -188,6 +198,13 @@ public int getArgon2Parallelism() {
         return argon2_parallelism;
     }
 
+    public String getFirebase_password_hashing_signer_key() {
+        if (firebase_password_hashing_signer_key == null) {
+            throw new QuitProgramException("'firebase_password_hashing_signer_key' cannot be null");
```

Comment:
this is bad..

Reply:
Just throw a new IllegalStateException -> which should yield a 500 error in sign in request (add a test for this as well)